### DISCUSSION TITLE
SmallRye AMQP concurrent message consuming

### DISCUSSION THREAD

**Discussion by serhiynovos** [quarkusio/quarkus#29092] [Q&A]
Is the any way how to configure amqp connector to consume messages from queue concurrently. For example one Quarkus app instance can process a few messages simultaneously. I saw Kafka connector has this capability but not AMQP

Thanks

[a comment by an automated account is not shown]

**Comment by ozangunalp** [comment]
By default reactive messaging messages are dispatched sequentially. You can check out the https://smallrye.io/smallrye-reactive-messaging/3.21.0/concepts/blocking/ annotation, in which you can configure unordered worker pool with concurrency.
